src/cursor.c

```diff
@@ -336,10 +324,25 @@ cursor_update_common(struct server *server, struct wlr_scene_node *node,
 		 * set the cursor image ourselves.
 		 */
 		wlr_seat_pointer_notify_clear_focus(wlr_seat);
-		set_server_cursor(seat, view_area);
+		set_server_cursor(seat, ctx->type);
 	}
 }
 
+static uint32_t determine_resize_edges(struct wlr_cursor *cursor,
```

Comment:
Style nitpick:
```c
static uint32_t
determine_resize_edges(..)
```

I personally would prefer `enum wlr_edges` as return value but I am not sure if the compiler likes the binary ORed together variants for that.

Reply:
I'll fix the formatting. `ssd_resize_edges()` already returns a `uint32_t` so I guess we should be consistent with that.